src/App.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import {BrowserRouter as Router} from 'react-router-dom';
+import { Switch, Route, Redirect } from 'react-router-dom';
+import { Provider} from 'react-redux';
+
+import store from './store';
+import ROUTES from './routes';
+import Header from './components/presentational/Header';
+import { Container } from './components/presentational/Layout';
+
+import './App.style'
+
+class App extends React.Component {
+  public render() {
+    return (
+      <Provider store={store}>
+        <Router>
+          <Container>
+            <Header />
+            <section>
+            <Switch>
+              {ROUTES.map(({path, component, exact}) =>
```

Comment:
For this specific case where we have few routes that are at the same level the benefit may not be that obvious, but remember that the mindset of React-router 4 is making routing dynamic instead of static. When defining your routes on a separate file you are still approaching this in a static way.

Reply:
Alright, should I move them to this single file?
Although, the dynamic routing could still be used in any other file connecting with router right?
I'm used to use single-file definitions for routes at the same level and use multiple files (dynamic) for nested routes or with a different context.